### PR TITLE
Refactoring import

### PR DESCRIPTION
**Pull request by adeolaawoyemi**
* Separated the import scripts for items and users so they can be run individually
* Updated the db access `config.js` file to use same way as import scripts
* added info to README about configuring db using `.env` file
* Added `process.exit()`s to import scripts so that terminal doesn't keep hanging even when jobs are done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+
+### MongoDB
+
+We have a cloud-hosted database in MongoDB Atlas. However, if you prefer using
+your own local copy, you can follow the instructions [here](https://zellwk.com/blog/install-mongodb/)
+to install your own.
+
+### Environment variables
+
+We use the dotenv package to to manege environment variables needed for things
```
Comment (BruceBGordon) [inline README.md:16]: man*a*ge

File: README.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+like database connection details.
+
+To setup conneting to the database, copy the file `.env.template` to `.env`
```
Comment (BruceBGordon) [inline README.md:19]: conne*c*ting

File: README.md [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+and add the values from the fields. This will allow the scripts that need
+to connect to the datase get the right connection info.
```
Comment (BruceBGordon) [inline README.md:21]: data*ba*se  ... *to* get the right connection

**Review by BruceBGordon — CHANGES_REQUESTED** [review]

**Review by yahoo-che — APPROVED** [review]

**Comment by adeolaawoyemi** [comment]
@BruceBGordon please approve this PR as well since you requested changes to the README that I've done.

**Review by BruceBGordon — APPROVED** [review]

**Pull request merged**